portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -105,6 +110,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentEntity.setProperty("message", comment);
     commentEntity.setProperty("name", name);   
     commentEntity.setProperty("picture", pictureURL);
+    commentEntity.setProperty("postNumber", postNumber);
+    postNumber++;
```

Comment:
This would be incompatible with concurrent requests or parallel instances of your server (which may or may not be enabled in AppEngine). I think you should look into using [entity keys](https://cloud.google.com/appengine/docs/standard/java/datastore/creating-entity-keys) instead.

Reply:
Does this mean I would still use postNumber but add it as a key instead of a property?

Reply:
Yep